### PR TITLE
Fix: docker release

### PR DESCRIPTION
**Pull request by lciolecki**

### CODE REVIEW

File: nussknacker-dist/src/universal/conf/docker-application.conf [hunk 1]
```diff
@@ -81,7 +81,8 @@ processConfig {
   }
   restartInterval: "10s"
   kafka = {
-    kafkaAddress = "kafka:9092"
+    zkAddress: "zookeeper:2181"
```
Comment (mproch) [inline nussknacker-dist/src/universal/conf/docker-application.conf]: do we need it?

File: nussknacker-dist/src/universal/conf/docker-application.conf [hunk 2]
```diff
@@ -20,15 +20,15 @@ authentication: {
...
-  headers {
-    Accept: "application/json"
+    grant_type: ${?OAUTH2_GRANT_TYPE]
   }
   authorizeParams {
-    response_type: "code"
+    response_type: ${?OAUTH2_RESPONSE_TYPE}
+    scope: ${?OAUTH2_SCOPE}
+  }
+  headers {
```
Comment (mproch) [inline nussknacker-dist/src/universal/conf/docker-application.conf]: maybe we should have defaults here? i Accept, response_type etc.?

Comment (lciolecki) [reply nussknacker-dist/src/universal/conf/docker-application.conf]: Hmm, you are right. I will add default at docker.

File: nussknacker-dist/src/universal/conf/docker-application.conf (hunk 1 above)
Comment (lciolecki) [reply nussknacker-dist/src/universal/conf/docker-application.conf]: Removed.

**Pull request merged**